### PR TITLE
feat: add an isActive prop to rentals

### PR DESCRIPTION
**Pull request by nicosantangelo**

### CODE REVIEW

File: src/handlers/rentals.ts [hunk 1]
```diff
@@ -9,8 +9,12 @@ import { OperatorUpdated, AssetRented, AssetClaimed } from '../entities/Rentals/
 export function handleAssetRented(event: AssetRented): void {
   let contractAddress = event.params._contractAddress.toHexString()
   let tokenId = event.params._tokenId
+
+  let currentRentalIndex = getRentalsCount(contractAddress, tokenId)
   let rentalIndex = getRentalsNextCount(contractAddress, tokenId)
```
Comment (fzavalia) [inline src/handlers/rentals.ts:14]: maybe rename to newRentalIndex

File: schema.graphql [hunk 2]
```diff
@@ -13,6 +13,7 @@ type Rental @entity {
   sender: String!
   ownerHasClaimedAsset: Boolean!
   isExtension: Boolean!
+  isActive: Boolean!
```
Comment (fzavalia) [inline schema.graphql:16]: maybe rename to isCurrent

**Review by fzavalia — APPROVED** [review]
Added some comments but are optional

File: src/handlers/rentals.ts (hunk 1 above)
Comment (nicosantangelo) [reply src/handlers/rentals.ts:14]: you mean `rentalIndex` right? good point!

**Pull request merged**